tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/FunctionDesc.java

```diff
@@ -167,6 +171,13 @@ public boolean equals(Object obj) {
     }
     return false;
   }
+
+  public boolean equalsSignature(Object obj) {
```

Comment:
Maybe this method is used to check the real equality between functions. If so, please fix the below code. In the below, function signature is compared with function desc.